packages/composite-checkout/src/lib/payment-methods/stripe-credit-card-fields.js

```diff
@@ -281,7 +281,7 @@ const StripeFields = styled.div`
 const CreditCardFieldsWrapper = styled.div`
 	padding: 16px;
 	position: relative;
-	display: ${( props ) => ( props.isLoaded ? 'block' : 'none') };
+	display: ${( props ) => ( props.isLoaded ? 'block' : 'none' )};
```

Comment:
I would've expected space in here: `${( props ) => ( props.isLoaded ? 'block' : 'none') }`